mangaki/mangaki/templates/fix/nsfw_grid.html

```diff
@@ -0,0 +1,137 @@
+        <div style="margin: 10px 0;">
+            {% include "mangaki/work_rating.html" with category=suggestion.work.category.slug work=suggestion.work bypass_nsfw_settings=True %}
+
+            {% if request.user.is_authenticated %}
+            <div class="nsfw-form">
+                <form action="{% url "update-evidence" %}?next={{ request.get_full_path }}" method="POST" class="form-group" style="margin-top: 10px;">
+                    {% csrf_token %}
+                    <input type="hidden" name="suggestion" value="{{ suggestion.pk }}">
+                    <button style="width: 100%;" class="btn {{ says_is_nsfw|yesno:'btn-success,btn-success,btn-default' }}"
+                            type="submit" name="agrees" value="{{ supposed_nsfw|yesno:'True,False' }}"
+                            {% if says_is_nsfw is not None %}disabled{% endif %}>
+                            C'est NSFW !
+                    </button>
+                </form>
+
+                <div class="small text-muted text-right">
+                    Signalée <em>{{ supposed_nsfw|yesno:'NSFW,non NSFW' }}</em> par
+                    <a href="{% url "profile" suggestion.user %}">{{ suggestion.user.username }}</a>.
+                </div>
+            </div>
+
+            <div class="nsfw-form-post text-muted small text-center" style="margin: 20px 0 0; display: none;">
+                Merci de votre vote.<br/>
+                Un administrateur interviendra aussi rapidement que possible !
+            </div>
+            {% endif %}
+        </div>
+
+        {% endfor %}
+        </div>
+        </div>
+
+        {% include "pagination.html" with page_obj=suggestions paginator=suggestions.paginator nsfw_ok=True %}
+    {% endif %}
+
+{% else %}
+
+    <div class="alert alert-warning text-center">
+        <strong>Aucun résultat à afficher :</strong> soit aucune œuvre n'a été marquée NSFW ou non NSFW, soit vous avez déjà donné votre avis.
+    </div>
+
+{% endif %}
+
+{% endblock %}
+
+{% block extrajs %}
+<script>
+    $('.nsfw-form form').submit(function(e) {
+        e.preventDefault();
+    });
+
+    $("[name='agrees']").on("click", function() {
+        var agrees_button = $(this);
+        var suggestion_id = $(this).closest('form').find("[name='suggestion']").val();
+
+        $.post(Urls['update-evidence'](), {
+            agrees: $(this).val(),
+            suggestion: suggestion_id,
+            success: function(data) {
+                agrees_button.closest('.nsfw-form').fadeOut(100, function() {
+                    $(this).next('.nsfw-form-post').fadeIn(100);
+                });
+                agrees_button.prop('disabled', true);
+
+                $(".skip-form input[name='suggestion'][value="+suggestion_id+"]").next("input[name='agrees']").remove();
+                $(".skip-form input[name='suggestion'][value="+suggestion_id+"]").remove();
+
+                if ($(".skip-form input[name='suggestion']").length <= 0) {
```

Comment:
Looks like strange to test if length could be negative :p !
Is there any reason I am missing (or am I too old in jQuery) ?

Anyway, if not, it's preferable to just test for `=== 0` I guess.